src/vs/workbench/api/node/extHostExtensionService.ts

```diff
@@ -38,6 +35,9 @@ import { RemoteAuthorityResolverError, ExtensionExecutionContext } from 'vs/work
 import { IURITransformer } from 'vs/base/common/uriIpc';
 import { ResolvedAuthority, ResolvedOptions } from 'vs/platform/remote/common/remoteAuthorityResolver';
 
+import { AZDataNodeModuleFactory, SqlopsNodeModuleFactory } from 'sql/workbench/api/node/extHostRequireInterceptor'; // {{SQL CARBON EDIT}}
```

Comment:
Is this the new convention for adding single line {{SQL CARBON EDIT}}

Reply:
Having the edits on the same line reduces the change of merge conflicts